### DISCUSSION TITLE
makeAddr undeclared error

### DISCUSSION THREAD

**Discussion by kingmanas** [Cyfrin/foundry-full-course-cu#1166] [General]
![Screenshot 2023-12-03 180927](https://github.com/Cyfrin/foundry-full-course-f23/assets/110822154/201d0473-505b-43ef-9ee2-190ed8ff377a)

**Comment by 0xusmanf** [comment]
Make sure you are importing the `Test.sol` and inheriting it correctly in your test contract like in the example below.
```solidity
// SPDX-License-Identifier: MIT

pragma solidity 0.8.19;

import {Test} from "forge-std/Test.sol";

contract ExampleTest is Test {
    address user = makeAddr("User");
}
```
